packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -255,7 +258,15 @@ describe("SharedTree", () => {
 	});
 
 	it("handle in op", async () => {
-		const provider = await TestTreeProvider.create(2);
+		// TODO: schema should be specified to enable compressed encoding.
```

Comment:
will you file an item and associate this todo (and the other one) with it? e.g. "TODO: ADO#XXX: schema should..."

Reply:
Updated TODO comments with [ADO#7111](https://dev.azure.com/fluidframework/internal/_sprints/backlog/Team%20Taylor/internal/Team%20Taylor%20-%202024.Bubbletea?workitem=7111)